__mocks__/react-i18next.tsx

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-import { translate, translateWithFormatting } from 'utils/test/i18n';
+import { translate, translateWithFormatting } from '../utils/test';
```

Comment:
out of interest, can the mocks not make use of the path resolution?

Reply:
They can, but an IDE will complain that the path can't be resolved because it would only work when jest subs the file into the right place

Reply:
But this is only the case for mocking anything from `node_modules`. Mocks for our own stuff will be in a path where tsconfig.json exists